test/fuzz/README.md

```diff
@@ -1,7 +1,11 @@
-# Corpra for fuzz tests with fixed API calls scenarios
-These corpra contain UR API calls in a predefined order described below.
+# Corpora for fuzz tests with fixed API calls scenarios
```

Comment:
Do people actually use the word 'corpora'? :D I get that this is the right Latin-derived plural, but it just sounds odd to me.

Reply:
Hehe, I have no idea. It's the first time I see this word😄
Anyway, I don't think `Corpra` was a proper word - do you have a suggestion in mind?


// I found `corpora` used once, here: https://github.com/google/fuzzing/blob/master/tutorial/libFuzzerTutorial.md#other-fuzzing-engines

Reply:
And also in the libFuzzer docs here (also once): https://llvm.org/docs/LibFuzzer.html - `corpora` is correct.